x-pack/plugins/data_quality/common/index.ts

```diff
@@ -9,7 +9,7 @@ import { i18n } from '@kbn/i18n';
 
 export const PLUGIN_ID = 'data_quality';
 export const PLUGIN_NAME = i18n.translate('xpack.dataQuality.name', {
-  defaultMessage: 'Logs data quality',
+  defaultMessage: 'Dataset Quality',
```

Comment:
This has probably already been discussed elsewhere, but I couldn't help noticing that this terminology diverges from the [brand guidelines](https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide) (which is what we generally follow in the docs too):  "data set" instead of "dataset".

Reply:
I've checked and in Kibana codebase, the user facing copies use "Data set". Whereas there are documentation pages which use "Dataset" e.g. [this](https://www.elastic.co/guide/en/elasticsearch/reference/8.13/getting-started.html) and [this](https://www.elastic.co/guide/en/elasticsearch/reference/8.13/xpack-ccr.html).

@ruflin @mdbirnstiehl   do you have any input about this?

Writing style guide [mentions](https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide/t/889c93):
![image](https://github.com/elastic/kibana/assets/2748376/98cd5dd1-8d26-402d-80ee-15f7133fe65f)

Reply:
I would go with what Lisa suggests and use data set and be consistent with the codebase.

Reply:
With "Data Set Quality", the UI would like:
![image](https://github.com/elastic/kibana/assets/2748376/99d3c74b-c620-44c6-aeb9-8e15b7daa3fd)

Reply:
LGTM